packages/node_modules/@webex/internal-plugin-encryption/src/encryption.js

```diff
@@ -113,17 +101,36 @@ const Encryption = WebexPlugin.extend({
       }
     })
       .then((res) => {
-        const url = res.body.endpoints[scr.loc];
+        const url = scr.loc;
 
-        if (!url) {
-          this.logger.warn('encryption: could not determine download url for `scr.loc`; attempting to download `scr.loc` directly');
+        return _this2._callDownloadUrl(url, scr, res);
```

Comment:
`_this2` should be able to be just `this`, mentioned [here](https://github.com/webex/webex-js-sdk/pull/1377/files#diff-22291d397df84ad20147dd45e8209ce4R83).